CODE_OF_CONDUCT.md

```diff
@@ -49,9 +48,9 @@ Project maintainers have the right and responsibility to remove, edit, or
 This Code of Conduct applies both within project spaces and in public spaces
  when an individual is representing the project or its community. Examples of
  representing a project or community include using an official project e-mail
- address, posting via an official social media account, or acting as
+ address, posting via an official social media account or acting as
```

Comment:
This is the [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/). I'd like to keep it.

Reply:
Okay, sir.